ext/websocket/lib.rs

```diff
@@ -514,18 +523,47 @@ pub async fn op_ws_close(
   Ok(())
 }
 
+#[op]
+pub fn op_ws_get_buffer(state: &mut OpState, rid: ResourceId) -> ZeroCopyBuf {
+  let resource = state.resource_table.get::<ServerWebSocket>(rid).unwrap();
+  resource.buffer.take().unwrap().into()
+}
+
+#[op]
+pub fn op_ws_get_buffer_as_string(
+  state: &mut OpState,
+  rid: ResourceId,
+) -> String {
+  let resource = state.resource_table.get::<ServerWebSocket>(rid).unwrap();
+  // TODO(mmastrac): We won't panic on a bad string, but we return an empty one.
```

Comment:
Is there a reason not to throw error here?

Reply:
Previously we were panicking and killing Deno, empty string is better but ideally we should close the WebSocket. We don't have a test for this right now.